src/pages/LoginRegister/LoginPage.css

```diff
@@ -0,0 +1,196 @@
+@import url("https://fonts.googleapis.com/css?family=Montserrat:400,800");
+
+* {
+  box-sizing: border-box;
+}
+
+.LoginRegisterForm_body {
+  background: #e5e5e5;
```

Comment:
Please remove the background color here and move it instead to the App.css file by targeting the body.

Reply:
okay! moved!